utilities/rocAL/rocal_external_source/README.md

```diff
@@ -0,0 +1,25 @@
+# rocal_external_source application
+This application demonstrates a basic usage of rocAL's C API to load images from external source and add augmentations and and displays the output images.
+
+## Build Instructions
+
+### Pre-requisites
+* Ubuntu Linux, [version `18.04` or later](https://www.microsoft.com/software-download/windows10)
+* rocAL library (Part of the MIVisionX toolkit)
+* [OpenCV 3.1](https://github.com/opencv/opencv/releases) or higher
+* Radeon Performance Primitives (RPP)
+
+### build
+  ````
+  export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/rocm/mivisionx/lib
+  export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/rocm/rpp/lib
+  mkdir build
+  cd build
+  cmake ../
+  make 
+  ````
+### running the application  
+  ````
+  rocal_external_source <image_dataset_folder/video_file> <processing_device=1/cpu=0>  decode_width decode_height batch_size gray_scale/rgb/rgbplanar display_on_off external_source_mode
```

Comment:
can we give video file also for external source?
if not please remove this from comments

Reply:
@swetha097 Please check this